enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,242 @@
+- A `EtcdBackup` CR is being installed on the cluster.
+- In a default config scenario, the use should not add any config.
+- A PVC and PV are being created to save the backup data reliably.
+
+
+### API Extensions
+- No API changes are required. 
+- See https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+
+### Implementation Details/Notes/Constraints [optional]
+- Need to agree on a default schedule.
+  - We can add default value annotation to current `Etcdbackup Spec`.
+- Need to agree on a default retention policy.
+  - We can add default value annotation to current `Etcdbackup Spec`.
+
+Example of default config
+
+```Go
+type EtcdBackupSpec struct {
+
+        // +kubebuilder:default:=0 */2 * * *
+	Schedule string `json:"schedule"`
+
+        // +kubebuilder:default:=daily
+	TimeZone string `json:"timeZone"`
+
+        // +kubebuilder:default:=RetentionNumber
+	RetentionPolicy RetentionPolicy `json:"retentionPolicy"`
+
+	// This field will be populated programmatically according to the infrastructure type.
+	// see discussion below
+	// +optional
+	PVCName string `json:"pvcName"`
+}
+```
+
+- For `PVCName`, we can populate it programmatically according to the underlying infra. See [Design Details](#design-details).
+
+### Risks and Mitigations
+
+When the backups are configured to be saved to a `local` type PV, the backups are all saved to a single master node where the PV is provisioned on the local disk.
+
+In the event of the node becoming inaccessible or unscheduled, the recurring backups would not be scheduled. The periodic backup config would have to be recreated or updated with a different PVC that allows for a new PV to be provisioned on a node that is healthy.
+
+If we were to use `localVolume`, we need to find a solution for balancing the backups across the master nodes, some ideas could be
+
+- create a PV for each master node using `localVolume` and the backup controller from CEO should take care of balancing the backups across the `available` and `healthy` volumes.
+- the controller should keep the most recent backup on a healthy node available for restoration.
+- the controller should skip an unhealthy master node from taking a backup
+
+
+### Drawbacks
+
+## Design Details
+
+- Add default values to current `Etcdbackup Spec` using annotation.
+- Maintain the reconciliation logic with Cluster-etcd-operator.
+
+### Initial Storage Design proposal
+- Several options exist for the default `PVCName`.
+  - Relying on `dynamic provisioning` is sufficient, however not an option for `SNO` or `BM` clusters.
+  - Utilising `local storage operator` is a proper solution, however installing a whole operator is too much overhead.
+  - The most viable solution to cover all OCP variants is to use `local volume`.
```

Comment:
You don't have to necessarily install local-storage-operator to use/consume local-storage. For simple one-off use cases like this, it may be possible that etcd-backup operator or something creates static-pv as documented - https://docs.openshift.com/container-platform/4.15/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html#local-create-cr-manual_persistent-storage-local and uses it to perform etcd backups.